tests/model/operation/transform/utils.js

```diff
@@ -0,0 +1,307 @@
+import ModelTestEditor from '@ckeditor/ckeditor5-core/tests/_utils/modeltesteditor';
+
+import BoldEditing from '@ckeditor/ckeditor5-basic-styles/src/bold/boldediting';
+import ListEditing from '@ckeditor/ckeditor5-list/src/listediting';
+import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
+import Typing from '@ckeditor/ckeditor5-typing/src/typing';
+import UndoEditing from '@ckeditor/ckeditor5-undo/src/undoediting';
+import BlockQuoteEditing from '@ckeditor/ckeditor5-block-quote/src/blockquoteediting';
+
+import { getData, setData, parse } from '../../../../src/dev-utils/model';
+import transform from '../../../../src/model/operation/transform';
+import Position from '../../../../src/model/position';
+import Range from '../../../../src/model/range';
+import OperationFactory from '../../../../src/model/operation/operationfactory';
+
+export class Client {
+	constructor( name ) {
+		this.editor = null;
+		this.document = null;
+		this.syncedVersion = 0;
+		this.orderNumber = null;
+		this.name = name;
+	}
+
+	init() {
+		return ModelTestEditor.create( {
+			plugins: [ Typing, Paragraph, BoldEditing, ListEditing, UndoEditing, BlockQuoteEditing ]
```

Comment:
Since you operate on the model, ignore schema, do you need to load these plugins?

Reply:
Not sure 🤔